shivakrishna/facebook_login.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#3b5998"
+    tools:context="com.example.root.facebook.MainActivity">
+
+<ImageView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:src="@drawable/fblogo"
+    android:layout_marginTop="70dp"
+    android:id="@+id/logo"/>
+    <TextView
```

Comment:
You don't have to test it on phone everytime. That's why we have AVDs i.e., Android Virtual Devices. It acts like your real phone so you can perform almost all tests there. Have you configured an AVD?